resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                    message: You cannot create or transfer a LISA account because the investor has failed a compliance check
+                prevAccntDoNoExists:
+                  description: The transferredFromAccountId and transferredFromLMRN given do not match an account on HMRC’s records
+                  value:
+                    code: PREVIOUS_INVESTOR_ACCOUNT_DOES_NOT_EXIST
+                    message: Previous Investor Account Doesn’t Exist
+                noTransferAccntData:
+                  description: You must give a transferredFromAccountId, transferredFromLMRN and transferInDate when the creationReason is transferred, current or previous year funds transferred
+                  value:
+                    code: TRANSFER_ACCOUNT_DATA_NOT_PROVIDED
+                    message: transfer account data not provided
+                transferAccntDataProvided:
+                  description: You must only give a transferredFromAccountId, transferredFromLMRN, and transferInDate when the creationReason is transferred, current or previous year funds transferred
+                  value:
+                    code: TRANSFER_ACCOUNT_DATA_PROVIDED
+                    message: transfer account data provided
+                accountClosed:
+                  $ref: '#/components/examples/accountClosed'
+                accountCancelled:
+                  $ref: '#/components/examples/accountCancelled'
+                accountVoid:
+                  $ref: '#/components/examples/accountVoid'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              example:
+                code: INVESTOR_ACCOUNT_ALREADY_EXISTS
+                message: This investor already has a LISA account
+                accountId: '1234567890'
+      deprecated: false
+      security:
+        - user-restricted:
+            - write:lisa
+  /lifetime-isa/manager/{lisaManagerReferenceNumber}/accounts/reinstate-account:
+    post:
+      tags:
+        - LISA Accounts
+      summary: Reinstate a LISA account
+      description: |
+        Re-open a LISA account that has been closed.
+        
+        ### Test data
+
+        <table style="font-family:Roboto, sans-serif; font-size:14px;">
+            <col width="25%">
+            <col width="35%">
+            <col width="40%">
+            <thead>
+                <tr>
+                    <th>Scenario</th>
+                    <th>Request Payload</th>
+                    <th>Response</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><p>Request with a valid payload, LISA Manager reference number and account ID</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks